docs/example_go.md

```diff
@@ -125,7 +125,7 @@ elapsed, then display this in both logs and HTML.
 Let's click the button on the `deploy` resource and see what happens!
 
 <figure>
-  <a class="is-image" href="https://cloud.tilt.dev/snapshot/AbakyOYLWWvW6J2BJFo=">
+  <a class="is-image" href="https://cloud.tilt.dev/snapshot/AaLb3ucLQ6BD9uBuppM=">
```

Comment:
```suggestion
  <a class="is-image" href="https://cloud.tilt.dev/snapshot/Ady3yOYL5yQatiGjVL0=">
```

here's the original snapshot for that image, cloud.tilt.dev FTW!

Reply:
🙌